socorro/unittest/external/fs/test_tar_crashstore.py

```diff
@@ -1,85 +1,143 @@
-from nose.tools import eq_, ok_, assert_raises
-from mock import Mock, patch
+from nose.tools import ok_, eq_
+from mock import Mock
 from datetime import datetime
 
 import json
+import tarfile
+import gzip
 
 from configman.dotdict import DotDict
 
 from socorro.external.fs.crashstorage import (
-    TarFileCrashStore,
+    TarFileWritingCrashStore,
+    TarFileSequentialReadingCrashStore
 )
-from socorro.external.crashstorage_base import CrashIDNotFound
+from socorro.external.crashstorage_base import Redactor
 from socorro.unittest.testbase import TestCase
 
-class TestTarCrashStorage(TestCase):
 
+#==============================================================================
+class TestTarFileWritingCrashStorage(TestCase):
+
+    #--------------------------------------------------------------------------
     def setUp(self):
-        super(TestTarCrashStorage, self).setUp()
+        super(TestTarFileWritingCrashStorage, self).setUp()
 
+    #--------------------------------------------------------------------------
     def _get_config(self):
         config = DotDict()
         config.logger = Mock()
         config.tarball_name = '/tmp/a_tarball_name.tar'
```

Comment:
The `/tmp` directory is bad to hardcode. It's much better to let the OS specify the temporary directory. E.g.

```python
config.tarball_name = os.path.join(tempfile.gettempdir(), 'a_tarball_name.tar')
```